python/lsst/sconsUtils/state.py

```diff
@@ -152,7 +152,7 @@ def _get_macosx_deployment_target_default():
     uname = os.uname()
     if uname.sysname == "Darwin" and uname.machine == "arm64":
         return "11.0"
-    return "10.9"
+    return "10.13"
```

Comment:
It seems that we probably should also use the env var if we know we are in a conda environment (since it is set by conda).